lib/ffmpeg/transcoder.rb

```diff
@@ -204,6 +211,12 @@ def transcode_movie
           raise Error, "Process hung. Full output: #{@output}"
         end
       end
+
+      FileUtils.cp(temp_output_file, @output_file)
```

Comment:
Do we need to write these interim paths at all to S3 since this is read by this job only?
Can it be written to `/tmp` and read from `/tmp`?

Pre-encoding w/ interim's happens on a per output basis, not per job. Meaning they wouldn't need to be shared AFAIK.

Reply:
Good catch, yeah that makes sense!